### PR TITLE
docs: command line

### PR DESCRIPTION
**Pull request by killme2008**
The reference of `greptime` command line.

### CODE REVIEW

**Review by apdong2022 — COMMENTED** [review]
can "start" be replaced by "deploy" "create" "initiate" etc? if not, then let's keep using it for all

File: docs/reference/command-lines.md [hunk 1]
```diff
@@ -1,3 +1,68 @@
 # Command lines
 
-TODO
+The command line `greptime` in pre-built binary can start/stop GreptimeDB and pass configuration options.
+
+
+`help` lists all commands and options of the `greptime`.
```
[the remark this answers is not in the document]
Comment (killme2008) [reply docs/reference/command-lines.md:6]: `greptime` is a command line tool, so it must be lowercase.

**Comment by killme2008** [comment]
> 

`Start` is better, it's more idiomatic in this case.

File: docs/reference/command-lines.md [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+
+OPTIONS:
+    -h, --help                     Print help information
+        --log-dir <LOG_DIR>        [default: /tmp/greptimedb/logs]
+        --log-level <LOG_LEVEL>    [default: info]
+
+SUBCOMMANDS:
+    datanode
+    frontend
+    help          Print this message or the help of the given subcommand(s)
```
[the remark this answers is not in the document]
Comment (killme2008) [reply docs/reference/command-lines.md:23]: This is the raw output of `greptime` command.

**Review by apdong2022 — APPROVED** [review]

**Pull request merged**